doc/adr/template.md

```diff
@@ -0,0 +1,72 @@
+# [short title of solved problem and solution]
+
+* Status: [proposed | rejected | accepted | deprecated | … | superseded by [ADR-0005](0005-example.md)] <!-- optional -->
+* Deciders: [list everyone involved in the decision] <!-- optional -->
```

Comment:
Even this: why spend time on documenting it if it's already embeded in the commits (and in that extend the PR number through which the commit was merged into the master)

Reply:
Hmm, but not everyone will commit. Ok the PR will show more info. I guess optional would be fine there

Reply:
I added deciders as optional again